doc/MANUAL.rst

```diff
@@ -327,6 +323,81 @@ glue. To do so use the ``-e`` switch. It will turn the python code into c++
 code you can call from a C++ code. In that case there is **no** need for a
 particular Pythran specification.
 
+Customizing Your ``.pythranrc``
+-------------------------------
+
+Pythran checks for a file named ``.pythranrc`` and loads additionnal
+configuration info from it. Here are a few tricks!
+
+``[sys]`` and ``[user]``
+========================
+
+These sections contains compiler flags configuration:
+
+:``cxx``:
+
+    Path to the compiler to use
+
+:``cppflags``:
+
+    Additionnal preprocessor flags (``-I``, ``-D``, ``-U``). Pythran is
+    sensible to ``-DUSE_BOOST_SIMD`` and ``-DUSE_GMP``. The former turns on
+    Boost.simd vectorization and the latter turns on Multi-Precision Integers.
```

Comment:
As DUSE_GMP is the default, can we disable this feature?
Also, should we document here : PYTHRAN_OPENMP_MIN_ITERATION_COUNT ?